ion-schema/src/ion_schema_version.rs

```diff
@@ -70,10 +69,15 @@ impl<T, V: IslVersion> Versioned<T, V> {
     }
 
     /// Consumes this [`Versioned`] instance, return the value contained within.
-    pub(crate) fn into_inner(this: Versioned<T, V>) -> T {
-        this.0
+    pub(crate) fn into_inner(self) -> T {
+        self.0
```

Comment:
🗺️ Now, instead of `Versioned::into_inner(value)`, you can call `value.into_inner()`.